cdap-ui/app/features/tracker/routes.js

```diff
@@ -161,6 +161,28 @@ angular.module(PKG.name + '.feature.tracker')
                 });
 
               return defer.promise;
+            },
+            rSystemTags: function ($q, myTrackerApi, $stateParams) {
+              if ($stateParams.entityType !== 'datasets') {
+                return null;
+              }
+
+              let defer = $q.defer();
+
+              let params = {
+                namespace: $stateParams.namespace,
+                entityType: $stateParams.entityType,
+                entityId: $stateParams.entityId
+              };
+              myTrackerApi.getSystemTags(params)
+                .$promise
+                .then( (res) => {
```

Comment:
We could simplify this to,

```
myTrackerApi.getSystemTags(params)
  .$promise
  .then(defer.resolve, defer.reject);
```
It would be same and concise.